test/e2e/config/nutanix.yaml

```diff
@@ -169,8 +170,10 @@ variables:
   WORKER_MACHINE_COUNT: 3
   NUTANIX_PRISM_ELEMENT_CLUSTER_NAME: ""
   NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME: "ubuntu-2004-kube-v1.23.8.qcow2"
-  CONTROL_PLANE_MACHINE_TEMPLATE_UPGRADE_TO: "ubuntu-2004-kube-v1.24.10.qcow2"
-  WORKERS_MACHINE_TEMPLATE_UPGRADE_TO: "ubuntu-2004-kube-v1.24.10.qcow2"
+  CONTROL_PLANE_MACHINE_TEMPLATE_UPGRADE_TO: "cluster-upgrade-conformance"
+  WORKERS_MACHINE_TEMPLATE_UPGRADE_TO: "cluster-upgrade-conformance"
+  NUTANIX_MACHINE_TEMPLATE_IMAGE_UPGRADE_TO: "ubuntu-2004-kube-v1.24.10.qcow2"
```

Comment:
lets document in the code to refer to these image name variables to verify if present as had to dig thru them to find which variables are really used or need to set

Reply:
also need to think how to automate this on prow as that pc wont have these images